README.md

```diff
@@ -69,9 +70,41 @@ If you want to run the latest development version from git source, you need to c
 git clone https://github.com/nextcloud/polls.git
 ```
 
-* Install dev environment with ```make setup-dev```
+* Install dev environment with ```make setup-dev``` or
+* install runtime environment with ```make setup-build```
 * Compile javascript with ```npm run build```
 * Run a complete build with ```make appstore``` (Find the output in the build directory)
+* call `occ app:enable polls` to enable Polls
+
+### Installation variants
```

Comment:
The header level is the same as the next subsections' header level; should they be different?